Mathlib/Tactic/Basic.lean

```diff
@@ -260,6 +260,12 @@ elab "fapply " e:term : tactic =>
 elab "eapply " e:term : tactic =>
   evalApplyLikeTactic (Meta.apply (cfg := {newGoals := ApplyNewGoals.nonDependentOnly})) e
 
+/--
+Tries to solve the goal using a canonical proof of `true` or the `reflexivity` tactic.
+-/
```

Comment:
```suggestion
/--
Tries to solve the goal using a canonical proof of `true` or the `reflexivity` tactic.
Unlike `trivial` or `trivial'`, does not use the `contradiction` tactic.
-/
```

Let's try to preserve existing documentation: tactic documentation is all being ported by hand.